pylint/checkers/exceptions.py

```diff
@@ -550,13 +561,19 @@ def visit_tryexcept(self, node: nodes.TryExcept) -> None:
                                 "bad-except-order", node=handler.type, args=msg
                             )
                     if (
-                        exception.name in self.linter.config.overgeneral_exceptions
+                        exception.qname() in self.linter.config.overgeneral_exceptions
+                        # TODO: 3.0: not a qualified name, deprecated
+                        or "." not in exception.name
+                        and exception.name in self.linter.config.overgeneral_exceptions
                         and exception.root().name == utils.EXCEPTIONS_MODULE
-                        and not _is_raising(handler.body)
                     ):
-                        self.add_message(
-                            "broad-except", args=exception.name, node=handler.type
-                        )
+                        if not _is_raising(handler.body):
```

Comment:
Why move this condition from the previous if ?

Reply:
I think I was trying to avoid the line length limit but it doesn't seem like that was necessary, fixed.